src/test/deserialization_tests.cpp

```diff
@@ -85,6 +88,70 @@ class TensorflowGRPCPredict : public ::testing::TestWithParam<ovms::Precision> {
 class DeserializeTFTensorProto : public TensorflowGRPCPredict {};
 class DeserializeTFTensorProtoNegative : public TensorflowGRPCPredict {};
 
+class CAPIPredict : public ::testing::TestWithParam<ovms::Precision> {
+protected:
+    void SetUp() override {
+        auto precision = ovms::Precision::FP32;
+        tensorMap[tensorName] = std::make_shared<ovms::TensorInfo>(
+            tensorName,
+            precision,
+            shape_t{1, 3},
```

Comment:
Shape mismatch - tensor map contains shape [1, 3] but in line 103 you declare [1, 10]. Why does the test pass?